pkg/csource/options.go

```diff
@@ -198,3 +207,81 @@ func DeserializeOptions(data []byte) (Options, error) {
 	}
 	return opts, err
 }
+
+type Feature struct {
+	Description string
+	Enabled     bool
+}
+
+type Features map[string]Feature
+
+func defaultFeatures(value bool) Features {
+	return map[string]Feature{
+		"tun":         {"setup and use /dev/tun for packet injection", value},
+		"net_dev":     {"setup more network devices for testing", value},
+		"net_reset":   {"reset network namespace between programs", value},
+		"cgroups":     {"setup cgroups for testing", value},
+		"binfmt_misc": {"setup binfmt_misc for testing", value},
+	}
+}
+
+func ParseFeaturesFlags(enable string, disable string, defaultValue bool) (Features, error) {
+	if enable == "none" && disable == "none" {
+		return defaultFeatures(defaultValue), nil
+	}
+	if enable != "none" && disable != "none" {
+		return nil, fmt.Errorf("can't use -enable and -disable flags at the same time")
+	}
+	if enable == "all" || disable == "" {
+		return enableAllFeatures(defaultFeatures(defaultValue)), nil
+	}
+	if disable == "all" || enable == "" {
```

Comment:
OK, I see there 3 special values: "all", "none", "" and e..g disable="all" is the same as enable="".

Reply:
Correct.